mmsplice/__init__.py

```diff
@@ -8,9 +8,7 @@
 
 from keras.models import load_model
 import mmsplice.generic
-import mmsplice.interval_tree
 import mmsplice.vcf_dataloader
-import mmsplice.exon_dataloader
```

Comment:
do we still have "exon_dataloader" somewhere? It is useful in many cases.

Reply:
`exon_dataloader` will be implemented in a later release. Please see #17.